.buildkite/release.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - label: Release
+    agents:
+      image: golang:${GO_VERSION}
```

Comment:
Should we define a default for this within the pipeline, otherwise I think we can set something like this at the top of the pipeline so the build fails if the version isn't set:

```yaml
env:
  GO_VERSION: ${GO_VERSION?}
```

Reply:
Added a hook with a default for the var.